server/src/lib/dokku/apps/logs.ts

```diff
@@ -1,10 +1,20 @@
 import NodeSsh from 'node-ssh';
 
 const parseLogsCommand = (commandResult: string) => {
-  const allLogs = commandResult.split('\u001b');
-  //We remove redundnat chars from good log & pick only 3rd item in array
-  //which is a good log
-  const logs = allLogs[2].slice(4);
+  // We split logs into array by new line
+  const allLogs = commandResult.split('\n');
+  const logs = [];
+  allLogs.map((log) => {
```

Comment:
as you are using .map either you have to do `logs = allLogs.map((log) => {` or use a forEach loop